index.js

```diff
@@ -7,10 +7,21 @@
 
 'use strict'
 
-var Q = require('q')
-var deep = require('deep-aplus')(Q.Promise)
+var Promise = global.Promise
+var deepAplus = require('deep-aplus')
+var deep = deepAplus(Promise)
```

Comment:
I would omit this line. `deepAplus` should be called when it is clear which promise-constructor to use.
See comment in line 45